src/utils/post-utils/post-services.js

```diff
@@ -108,4 +108,51 @@ export const editPost=createAsyncThunk('posts/editPost',async(postData)=>{
     } catch (error) {
         console.log('error occured',error)
     }
+})
+
+export const addToBookmarks=createAsyncThunk("posts/addToBookmarks",async(postId)=>{
+    const encodedToken=localStorage.getItem('token')
+    try {
+        const res=await axios({
+            method:"POST",
+            url:`/api/users/bookmark/${postId}`,
+            headers:{authorization:encodedToken},
+        })
+        if(res.status===200){
+            return res.data.bookmarks
+        }
+    } catch (error) {
+        console.log("error occured",error)
+    }
+})
+export const deleteFromBookmarks=createAsyncThunk("posts/deleteFromBookmarks",async(postId)=>{
+    const encodedToken=localStorage.getItem('token')
+    try {
+        const res=await axios({
+            method:"POST",
+            url:`/api/users/remove-bookmark/${postId}`,
+            headers:{authorization:encodedToken},
+        })
+        if(res.status===200){
+            return res.data.bookmarks
+        }
+    } catch (error) {
+        console.log("error occured",error)
```

Comment:
Can use console.error to get error messages on the console helps in tracking and differentiating down further.